pynamodb/attributes.py

```diff
@@ -218,7 +218,7 @@ def __init__(self, **attributes):
         self._set_attributes(**attributes)
 
     @classmethod
-    def _get_attributes(cls):
+    def get_attributes(cls):
```

Comment:
while this isn't a public method i'm sure there's a lot of code calling it directly

should we keep the private signature that just calls the public method and mark it as deprecated?

Reply:
You and your backwards-compatible API design, smh.

Yeah, sounds good 👍 .